content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,827 @@
+
+The applicant flow explained in this document is the default flow in Odoo, and goes through the
+applicant flow when using the *Recruitment* application's default configuration. The applicant flow
+is able to be modified to suit the specific recruitment flow for any business.
+
+.. note::
+   The applicant flow with all its stages are universal and applies to all job positions, unless
+   specified. :ref:`A specific stage can be configured <stages>` to be job-specific, meaning that
+   specific stage is only visible for that specific job position. Otherwise, if a new stage is
+   created, or an existing stage is modified, those changes will be visible on all job positions.
+
+Kanban view
 ===========
-Recruitment
-===========
+
+To access the kanban view for a job position, navigate to the main :menuselection:`Recruitment app`
+dashboard, which is the default view when opening the application. All job positions appear on the
+main dashboard. Click the :guilabel:`New Applications` smart button on a job position card to
+navigate to the kanban view for all the applicants for that particular job position.
+
+.. image:: recruitment/new-applicants-button.png
+   :align: center
+   :alt: Main dashboard view of job position card, showing new applications button.
+
+Inside the job application, the kanban stages appear, with all the applicants populated in their
+respective columns, indicating what stage they are currently in. There are five default stages in
+Odoo:
+
+- :ref:`Initial Qualification <initial>`
+- :ref:`First Interview <interview>`
+- :ref:`Second Interview <second>`
+- :ref:`Contract Proposal <contract>`
+- :ref:`Contract Signed <signed>`
+
+The last column, :guilabel:`Contract Signed`, is folded by default, meaning that the column appears
+in gray and the applicants in it are hidden from view. To expand the folded stage and view the
+applicant cards for that column, click anywhere on the thin gray column that says the stage name and
+the column expands, revealing the applicants.
+
+.. image:: recruitment/stages.png
+   :align: center
+   :alt: Expand a folded column by clicking on it in the kanban view.
+
+Each stage has a color-coded bar beneath the stage name, giving information about the applicant's
+status in the stage. The colors are:
+
+- :guilabel:`Green`: There are applicants in the stage with an activity that is either already
+  scheduled or needs to be scheduled in the future (such as a phone call or interview), according to
+  the parameters for that particular stage.
+- :guilabel:`Yellow`: There are applicants in the stage with an activity either scheduled for today
+  or due to be scheduled today.
+- :guilabel:`Red`: There are applicants in the stage with overdue activities which need to be
+  scheduled.
+- :guilabel:`Gray`: There are either no activities currently scheduled and no future activities that
+  are required to be scheduled, or there are no applicants currently in the stage.
+
+.. _stages:
```

Comment:
I didn't realize there was a naming convention for the anchors- I was just so thrilled when I was able to do them, and in this one I went HAM! I have been using them more and more, since I really like chopping up the sections, and LOVE it if I (or a reader) can just to a corresponding spot. I adore anchors =) 

I will be sure to do very descriptive and appropriate anchors from now on. LMK if there is a limit, or something else I might be doing wrong by using so many.